### DISCUSSION TITLE
how to change the texts of the skip buttons

### DISCUSSION THREAD

**Discussion by danielpetinga** [ditdot-dev/vue-flow-form#289] [Q&A]
Hello everyone, everything good? I would like some help with the altomatic texts eg: how to change the texts of the skip buttons?
![aaaaaaa](https://github.com/ditdot-dev/vue-flow-form/assets/2942585/8cdfdd60-94ed-472e-924a-d63c18b4d328)

**Comment by spinn** [comment]
Hi - you need to override the "skip" variable when defining LanguageModel.

https://github.com/ditdot-dev/vue-flow-form/blob/a8f30b81543c2070589db49d9f326d2ec3287587/src/models/LanguageModel.js#L15
